src/dashboard/src/templates/transfer/grid.html

```diff
@@ -191,7 +191,7 @@
           </div>
           <div style="float:left;">
             <input type='text' id='transfer-accession-number'></input>
-            <div class="help-block">Accession no.</div>
+            <div class="help-block">TMS Object ID</div>
```

Comment:
Correct - this branch is a candidate to be the new MoMA deployed branch, and also a discussion place for how these things can be merged (or not) into qa. Accession number vs object id as slug for DIP upload is one of the things we'd have to work out.